config/locales/devise/en.yml

```diff
@@ -116,7 +116,7 @@ en:
       max_login_attempts_reached: >
         Your account is temporarily locked because you have entered the
         one-time passcode incorrectly too many times.
-      please_try_again: Please try again in %{time_remaining}.
+      please_try_again: Please try again in <span id="countdown">%{time_remaining}</span>.
```

Comment:
If we rename this key `please_try_again_html` we don't need to manually `.html_safe` this in the template.

Also, I consider it a best practie to keep HTML markup outside of i18n (because of human- and machine-introduced bugs that appear in translation pipelines) so I would recommend creating the `span` in the template:

```erb
  <%= t('devise.two_factor_authentication.please_try_again_html',
        time_remaining: content_tag(:span, decorated_user.lockout_time_remaining_in_words)) %>
```